app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<p> Project Name: <%= @project.name %></p>
+<p> Material: <%= @project.material %></p>
+<p> Challenge Theme: <%= @project.challenge.theme %></p>
+<p> Number of Contestants: <%= @project.contestants.count %>
```

Comment:
learned that .count in this situation is an AR method not an array method, so while I get the correct value, this logic should be moved to the model as a method 
`def number_of_contestants` 
`contestants.count`
`end`
which I would then call from the view. That way it's encapsulated (keeping DRY), abstracted, and can be tested more effectively. It's data logic (model) not presentation logic (view).

Reply:
I agree completely :) Thanks for making the comment for me!